scripts/start.sh

```diff
@@ -57,6 +57,15 @@ if [[ -f "${PG_ENV}" ]]; then
 fi
 
 echo "
+export HOST_BASE=$HOST_BASE
+export HOST_BUCKET=$HOST_BUCKET
+export DEFAULT_REGION=$DEFAULT_REGION
+export SSL_SECURE=$SSL_SECURE
+export ACCESS_KEY_ID=$ACCESS_KEY_ID
+export SECRET_ACCESS_KEY=$SECRET_ACCESS_KEY
+export EXTRA_CONF=$EXTRA_CONF
```

Comment:
Can EXTRA_CONF be multiline?
Not sure how it can fit inside this statement without heredoc.